Library/Formula/dasht.rb

```diff
@@ -0,0 +1,29 @@
+class Dasht < Formula
+  desc "Search API docs offline, in your terminal or browser"
+  homepage "https://sunaku.github.io/dasht"
+  url "https://github.com/sunaku/dasht/archive/v1.1.0.tar.gz"
+  sha256 "578bb8a1d576b5c487ca3d0c3113905ff68401530d4237f239b6c6aac62e1fbd"
+
+  bottle :unneeded
+
+  depends_on "sqlite"
+
+  def install
+    bin.install Dir["bin/*"]
+  end
+
+  def caveats
+    <<-EOS.undent
+      First, install some docsets using dasht-docsets-install(1)
```

Comment:
In that case I think we should remove it entirely; we shouldn't replicate the official documentation since our "version" will become out-of-date sooner or later.